components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineView.kt

```diff
@@ -118,6 +118,7 @@ class SystemEngineView @JvmOverloads constructor(
                 currentUrl = url
                 session?.internalNotifyObservers {
                     onLoadingStateChange(true)
+                    onLocationChange(it)
```

Comment:
This is good and the change I would have expected here :)

Reply:
👍